tests/upgrade/test_upgrade.py

```
@@ -0,0 +1,90 @@
+import time
+from common import *
```

Ruff

`from common import *` used; unable to detect undefined names

(F403)